Library/Homebrew/brew.sh

```diff
@@ -516,13 +516,13 @@ esac
 # - https://github.com/Homebrew/install/blob/HEAD/install.sh
 # and, if needed:
 # - MacOSVersion::SYMBOLS
-HOMEBREW_MACOS_NEWEST_UNSUPPORTED="15"
+HOMEBREW_MACOS_NEWEST_UNSUPPORTED="16"
```

Comment:
This will also make Sequoia CI run on every PR, which we probably don't want yet.

Changes will need to be made in `github_runner_matrix.rb`.

Reply:
We'll probably not merge this until there's good bottle coverage.

But we make need to make changes anyway as Intel will probably not be in sync with arm64 this year.

Reply:
> We'll probably not merge this until there's good bottle coverage.

We should merge this as soon as we're running Sequoia in CI.

It doesn't matter for end-users if we have zero Sequoia bottle coverage as we can still support it before that.